x/subscription/spec/02_state.md

```diff
@@ -0,0 +1,34 @@
+<!--
+order: 2
+-->
+
+# State
+
+The `x/subscription` stores the plans and subscriptions on chain:
+
+```golang
+type Plan struct {
+  id int,  // auto-increasing unique identifier
+  title string,
+  description string,
+  owner Address,  // beneficial owner of the plan
+  price Decimal,  // price of fiat currency show to user
+  price_currency String,  // fiet currency name
+  rate Decimal,  // fiat-token conversion rate, maintained by owner
+  
```

Comment:
rate to what ? what if e.g. one transfers in a stablecoin or a different asset over IBC and would want to create a subscription with that

Reply:
- added a `token_denomation` field.
- for stablecoin, merchant can maintain a near 1 conversion rate.

Reply:
removed rate, use fixed token denomination.